role-make-version-changelog.sh

```diff
@@ -306,9 +306,10 @@ You have three options:
             cat "$rel_notes_file" CHANGELOG.md > .tmp-changelog
         fi
         mv .tmp-changelog CHANGELOG.md
-        git add CHANGELOG.md
+        git show docs:latest/README.html > .README.html
```

Comment:
you can also use `git checkout $branchname -- $filename` to check the file out of the specified branch into the current branch, which will also do a `git add $filename`

Reply:
In that case, I'll need to rename `README.html` > `.README.html` after checkouting it, and here it's one line.